tests/utils/exec/idps.go

```diff
@@ -15,8 +15,11 @@ type IDPArgs struct {
 	Token         string        `json:"token,omitempty"`
 	OCMENV        string        `json:"ocm_environment,omitempty"`
 	URL           string        `json:"url,omitempty"`
+	CA            string        `json:"ca,omitempty"`
+	Insecure      bool          `json:"insecure,omitempty"`
 	MappingMethod string        `json:"mapping_method,omitempty"`
 	HtpasswdUsers []interface{} `json:"htpasswd_users,omitempty"`
+	// Attributes []string      `json:"attributes,omitempty"`
```

Comment:
Is it not relevant parameter? delete?

Reply:
It is not a mandatory field in order for the test to work, but I was thinking maybe in future tests
we will need to validate also all the optional fields for an idp. but anyway I'll delete it for this PR